packages/mobile/locales/en-US/nuxVerification2.json

```diff
@@ -50,6 +50,8 @@
     "body2": "You can skip verification for now and try again later.",
     "body1InsufficientBalance": "Your current balance is not high enough to complete the verification process.",
     "body2InsufficientBalance": "Please try again when you have added funds to your wallet.",
+    "body1SaltQuotaExceeded": "Your phone number lookup quota is used up",
```

Comment:
Looks like you need these in spanish as well

Reply:
Added them!